build.gradle.kts

```diff
@@ -25,6 +25,10 @@ dependencies {
   testImplementation("org.wiremock:wiremock:3.2.0")
   testImplementation("io.opentelemetry:opentelemetry-sdk-testing")
   testImplementation("com.google.code.gson:gson:2.10.1")
+  testImplementation("io.swagger.parser.v3:swagger-parser:2.1.18") {
+    exclude(group = "io.swagger.core.v3")
```

Comment:
swagger parser includes the old javax versions.  We were previously having to include
```
  testImplementation("javax.xml.bind:jaxb-api:2.3.1")
```
but after reading https://github.com/swagger-api/swagger-parser/issues/1981 found a slightly more verbose, but nicer way to get the parser working